opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -83,19 +97,37 @@ impl opentelemetry::logs::LoggerProvider for LoggerProvider {
 impl LoggerProvider {
     /// Create a new `LoggerProvider` builder.
     pub fn builder() -> Builder {
+        otel_info!(
+            target: otel_target!("logs"),
+            name: "logger_provider_builder"
+        );
         Builder::default()
     }
 
     pub(crate) fn log_processors(&self) -> &[Box<dyn LogProcessor>] {
+        otel_debug!(
+            target: otel_target!("logs"),
+            name: "logger_provider_log_processors",
+            processor_count= self.inner.processors.len()
+        );
         &self.inner.processors
     }
 
     pub(crate) fn resource(&self) -> &Resource {
+        otel_debug!(
+            target: otel_target!("logs"),
+            name: "logger_provider_resource",
+            resource_count= self.inner.resource.len()
+        );
         &self.inner.resource
     }
 
     /// Force flush all remaining logs in log processors and return results.
     pub fn force_flush(&self) -> Vec<LogResult<()>> {
+        otel_info!(
```

Comment:
Hmm how do we decide when to use `info` vs `debug` here?

Reply:
This is my take -  A good practice would be to use `otel_debug` for more granular logging, especially in hot-path scenarios. Meanwhile, `otel_info` can be reserved for higher-level events like SDK initialization, shutdown, and actions such as `force-flush`. For emitting logs, traces, and metrics, `otel_debug` is appropriate.